content/docs/command-reference/commit.md

```diff
@@ -21,9 +21,9 @@ positional arguments:
 The `dvc commit` command is useful for several scenarios, when data already
 tracked by DVC changes: when a [stage](/doc/command-reference/run) or
 [pipeline](/doc/command-reference/pipeline) is in development/experimentation;
-when manually editing or generating DVC <abbr>outputs</abbr>; or to force
-DVC-file updates without reproducing stages or pipelines. These scenarios are
-further detailed below.
+when manually editing or generating DVC <abbr>outputs</abbr>; or to force update
+the stages (or `.dvc` files) without reproducing the stages or pipelines. These
```

Comment:
Again, this command does not change stages at all. Feel free to contact me with questions @imhardikj .

Reply:
In context of this line, I have updated it to `to force update DVC-tracked files without...`.